.dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:18.6-slim
+
+LABEL maintainer="DocNetwork <webteam@docnetwork.org>"
```

Comment:
https://docs.docker.com/engine/reference/builder/#maintainer-deprecated

This label will work but all your doing is setting some metadata.  If you want to set it more formally you would use a label-schema.  

Ultimately this is an optional thing and doesn't really matter unless we are going to be sharing the container publicly.  You could remove it without any issue.

Reply:
I think using ```label-schema``` is a good idea since from what I know the repository is public and available to all.

Reply:
Just tried it, its saying ```label-schema`` is an ```Unknown Instruction```

Reply:
Hmm, what might cause that, @mmclane ?

Reply:
That should be: 
LABEL org.opencontainers.image.authors="webteam@docnetwork.org"

Reply:
yeah it worked